### DISCUSSION TITLE
[designer] Template expansion error: Cannot read properties of undefined (reading 'dataType')

### DISCUSSION THREAD

**Discussion by becory** [microsoft/AdaptiveCards#8297] [Q&A]
Hi, @anna-dingler 
I upgrade the version from 2.3.0 to 2.4.3.
found this error.

`Template expansion error: Cannot read properties of undefined (reading 'dataType')`
https://github.com/microsoft/AdaptiveCards/blob/45caa275735e210f27449570524d4f51a4aff40a/source/nodejs/adaptivecards-designer/src/card-designer-surface.ts#L319

I can't preview my sample data in the designer.
Is this a breaking change?
Could you tell me what can I do?

**Comment by anna-dingler** [comment]
Hi @becory,

Can you send me your card payload and the data?

**Reply by becory** [reply]
Hi @anna-dingler, I used the same payload and the data from the designer demo website.
https://adaptivecards.io/designer/

On the other hand, I found I can't drop any element from the left side menu in 2.4.3.

**Reply by anna-dingler** [reply]
@becory The drag and drop behavior should be working as expected. Are you editing in design mode?
